frontend/src/utils/index.js

```diff
@@ -410,8 +410,27 @@ export function isServiceAccount (username) {
   return startsWith(username, 'system:serviceaccount:')
 }
 
-export function isServiceAccountFromNamespace (username, namespace) {
-  return startsWith(username, `system:serviceaccount:${namespace}:`)
+export function isForeignServiceAccount (serviceAccount, namespace) {
+  if (serviceAccount && namespace) {
+    const { serviceAccountNamespace } = prefixedServiceAccountToComponents(serviceAccount)
+    if (serviceAccountNamespace && serviceAccountNamespace !== namespace) {
+      return true
+    }
+  }
+  return false
+}
+
+export function prefixedServiceAccountToComponents (serviceAccount) {
```

Comment:
Rename to `parseFullyQualifiedServiceAccountName`

Reply:
Actually, I had it like this before (fullQualified). However, I think this is not correct. I looked it up in the Kubernetes Documentation / Code and no one talks about full qualified there, they always refer to it as _service account prefix_. This is why I changed it to that name...

Reply:
We agreed to rename it to `ServiceAccountUsername`